cli/Cargo.toml

```diff
@@ -65,6 +65,9 @@ deno_ast = { version = "0.25.0", features = ["transpiling"] }
 birdcage = { version = "0.2.0" }
 libc = "0.2.135"
 
+[target.'cfg(windows)'.build-dependencies]
+winapi = { version = "0.3.9", features = ["winbase", "processenv"] }
```

Comment:
It's probably worth adding an issue (or at least a comment) explaining that this can be removed after the next `deno_io` release